src/fides/data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -217,4 +218,67 @@ privacy_notices:
   has_gpc_flag: True
   displayed_in_privacy_center: True
   displayed_in_overlay: True
+  displayed_in_api: False
+- name: Targeted Advertising
+  notice_key: targeted_advertising
+  id: [not sure what to put here]
+  description: We may use some of your information and share it with partners for the purpose of targeted advertising. You can learn more about what information is used for this purpose in our privacy notice.
+  internal_description: Targeted advertising" means displaying advertisements to a consumer where the advertisement is selected based on personal data obtained or inferred from that consumer's activities over time and across nonaffiliated Internet web sites or online applications to predict such consumer's preferences or interests.
+  regions:
+    - ca_ab
+    - ca_bc
+    - ca_mb
+    - ca_nb
+    - ca_nl
+    - ca_ns
+    - ca_on
+    - ca_pe
+    - ca_sk
+    - ca_nt
+    - ca_nu
+    - ca_yt
+  consent_mechanism: opt_out
+  data_uses:
+    - marketing.advertising.first_party.targeted
+    - marketing.advertising.third_party.targeted
+  enforcement_level: frontend
+  disabled: False
+  has_gpc_flag: True
+  displayed_in_privacy_center: True
+  displayed_in_overlay: True
+  displayed_in_api: False
+- name: Targeted Advertising and Marketing
+  notice_key: targeted_advertising_marketing
+  id: [not sure what to put here]
+  description: We may use some of your information and share it with partners for the purpose of marketing tracking and targeted advertising. You can learn more about what information is used for this purpose in our privacy notice.
+  internal_description: Advertising, targeting, or marketing cookies and services are technologies that tracks users' browsing behavior across websites. These technologies enable targeted advertising and marketing campaigns by collecting information about users' interests, preferences, and online activities. Marketing technologies typically require user consent under privacy regulations.
+  regions:
+    - ca_qc
+  consent_mechanism: opt_in
+  data_uses:
+    - marketing.advertising.first_party.targeted
+    - marketing.advertising.third_party.targeted
+  enforcement_level: frontend
+  disabled: False
+  has_gpc_flag: True
+  displayed_in_privacy_center: True
+  displayed_in_overlay: True
+  displayed_in_api: False
+- name: Functional and Essential
```

Comment:
Similar question as above - how can we make this easier for the customer to understand because once enabled, this'll appear right alongside the EU-style Functional & Essential notices which feels confusing.

Reply:
There are no other functional + essential notices that combine the two